projeto/controladores/deletarProduto.js

```diff
@@ -6,13 +6,28 @@ const readline = require("readline-sync");
 const { database } = require("../database");
 
 const deletarProduto = () => {
-  console.log("Digite um id de um produto para deletar");
+  console.log("Digite um id do produto que deseja deletar");
+
   const idSelecionado = readline.question();
-  console.log("Você selecionou o id: " + idSelecionado);
-  console.log("Dê enter para voltar");
-  readline.question();
-  console.clear();
-}
+  const newDatabase = database.find((item) => item.id === +idSelecionado);
+  console.table(newDatabase)
+  console.log("\n Se quer realmente excluir este produto, digite 1, ou outro valor para sair.")
+  const escolha = readline.question();
+  var newDatabase1 = database
+  var newDatabase2 = newDatabase1.filter(item => (item.id !== +idSelecionado));
+    
+  if(escolha === "1") {
+       console.log("O produto foi deletado");
+       console.table(newDatabase2);
+    }else { 
+      (escolha !== "1");
```

Comment:
o `else` diz respeito a tudo o que não tem relação com a condiçao do `if`.Como o seu `if` diz que só deve acontecer se a escolha for `1`, o `else` já entende que o que for diferente de `1` entra lá. Por isso, nao precisamos dessa linha 23. Faz sentido?